### DISCUSSION TITLE
Chrome

### DISCUSSION THREAD

**Discussion by PuzzleMonke** [legendary-gl/legendary#475] [Ideas]
I was wondering if you could make authentication happen on a Chrome tab. I have a school computer and want to play Epic Games, but when I write "legendary auth" it's blocked. On a chrome tab, I have a VPN that allows me to access blocked websites, including Epic Games.

**Comment by CommandMC** [comment]
`legendary auth --disable-webview` might help there

**Reply by PuzzleMonke** [reply]
Thank you, it worked!
